MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include VERSION
+include README.md
+recursive-include datasets/docs/*
```

Comment:
If you do decide to adopt `pyproject.toml` as your library building metadata choice, instead of `setup.py` (see the comment I left below), you can also remove this file and use the `include` feature of the `pyproject.toml` spec [link](https://python-poetry.org/docs/pyproject#include-and-exclude) 😎

Reply:
coming soon!